src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -1133,7 +1136,7 @@ public void setPinnedEndpointIds(Set<String> pinnedEndpointIds)
         if (!this.pinnedEndpointIds.equals(pinnedEndpointIds))
         {
             this.pinnedEndpointIds = new HashSet<>(pinnedEndpointIds);
-            update(lastEndpointOrdering, getAvailableBandwidth());
+            update();
```

Comment:
I moved this away from the single blanket update function because I found it confusing less explicit. Is there an issue with the updated scheme? Why go back?

Reply:
i found it confusing that an update() caller can set the lastendpointordering and then update() itself sets the lastendpointordering0; my idea is that update() computes a result based on the internal state, but it doesn't alter the state (other than storing the result of course)

Reply:
Ok, give me a chance to look on non-mobile and I can probably get a better picture. I'm thinking maybe we can at least rename update to something that reflects that better.